client/src/components/ui/Input/Input.style.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+interface InputProps {
+    fontSize?: "xs" | "sm" | "mds" | "md" | "mdl";
+    fontWeight?: "regular" | "light";
+    color?:
+    | "primary"
+    | "secondary"
+    | "placeholder";
+    background?: "primary";
+    width?: number;
+}
+
+export const Input = styled.input<InputProps>`
+    padding-bottom: 13px;
+    border-radius: 5px;
+    border: medium solid #000;
+    cursor: pointer;
+    font-size: ${({ theme, fontSize }) =>
+        fontSize ? theme.size.desktop[fontSize] : theme.size.desktop.mds}px;
+    font-weight: ${({ theme, fontWeight }) =>
+        fontWeight ? theme.weight[fontWeight] : theme.weight.regular};
+    background: ${({ theme, background }) =>
```

Comment:
If background always have to be "primary" I do not think it is need to do this function, instead we can just  leave :
`background: ${({ theme }) => theme.colors.input.primary};`
And remove it from props